### PR TITLE
Auto-gen update

### PR DESCRIPTION
**Pull request by jburel**
Add docker_virtual files to the list.

cc @hflynn

### CODE REVIEW

File: omero/autogen_docs [hunk 1]
```diff
@@ -51,6 +51,7 @@ for f in \
     omero-* \
     settings.env \
     setup_* \
+    docker_virtualenv*\
```
Comment (manics) [inline omero/autogen_docs:54]: Are these files only for Docker? If so why are they needed in the main docs?

Comment (jburel) [inline omero/autogen_docs:54]: We picked that term, I could not find a better one. I will review them post release. They are used in the server-centos6 installation

Comment (joshmoore) [inline omero/autogen_docs:54]: I think the idea is that these should be specific to the testing, and shouldn't be needed in the docs.

Comment (jburel) [inline omero/autogen_docs:54]: We have one link pointing to one the file (Apache24 support)
I can remove it and point to the omero-install instead so it is not included in the doc.

**Comment by jburel** [comment]
Closing this PR, I will edit the file in the documentation

**Pull request closed without merging**